src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/Migrations/SqliteDesignTimeJellyfinDbFactory.cs

```diff
@@ -19,7 +20,8 @@ public JellyfinDbContext CreateDbContext(string[] args)
             return new JellyfinDbContext(
                 optionsBuilder.Options,
                 NullLogger<JellyfinDbContext>.Instance,
-                new SqliteDatabaseProvider(null!, NullLogger<SqliteDatabaseProvider>.Instance));
+                new SqliteDatabaseProvider(null!, NullLogger<SqliteDatabaseProvider>.Instance),
+                new NoLockBehavior());
```

Comment:
Why all these new locking behaviour mechanism added to end up using no locking?

Reply:
this is just the design time context, its only used during the `dotnet migration add ` command and never on a user install so why bother locking here?